src/app/shared/form/builder/ds-dynamic-form-ui/models/ds-dynamic-input.model.ts

```diff
@@ -27,7 +27,7 @@ export interface DsDynamicInputModelConfig extends DynamicInputModelConfig {
   hasSelectableMetadata: boolean;
   metadataValue?: FormFieldMetadataValueObject;
   isModelOfInnerForm?: boolean;
-
+  hideRequiredHint?: boolean;
```

Comment:
probably it's better to have a more generic flag to hide a possible error message and not only the required error message

Reply:
Indeed, it was already implemented this way so I just changed the name to `hideErrorMessages` for clarity